src/scss/_mixins.scss

```diff
@@ -0,0 +1,108 @@
+
+/// Outputs all available features and styles for labels.
+///
+/// @param {List} $sizes - The label size variants to output styles for.
+/// @param {List} $states - The label state variants to output styles for.
+/// @output The output includes the `.o-labels` class definition as well as class definitions for every variant.
+/// @example scss - All label styles
+///   @include oLabels();
+/// @example scss - Base styles, big size variant, and gold tier state variant
+///   @include oLabels(('big'), ('tier-gold'));
+/// @access public
+@mixin oLabels($sizes: $_o-labels-sizes, $states: $_o-labels-states) {
```

Comment:
@notlee and I have chatted about file structure and such before  — we've agreed to put the primary mixin in the `main.scss` file, in order to make it easier to find and consistent across our components 😬